static/js/controllers/inbox.js

```diff
@@ -53,20 +53,39 @@ var feedback = require('../modules/feedback'),
 
       $scope.replicationStatus = {
         disabled: false,
-        lastSuccess: { }
+        lastSuccess: {},
+        current: 'unknown',
       };
-      DBSync(function(status) {
-        if (status.disabled) {
+      var SYNC_ICON = {
+        in_progress: 'fa-refresh',
+        not_required: 'fa-check',
+        required: 'fa-exclamation-triangle',
+        unknown: 'fa-question-circle',
+      };
+      DBSync(function(update) {
+        if (update.disabled) {
           $scope.replicationStatus.disabled = true;
           // admins have potentially too much data so bypass local pouch
           $log.debug('You have administrative privileges; not replicating');
           return;
         }
-        var now = Date.now();
-        var last = $scope.replicationStatus.lastSuccess[status.direction];
-        $scope.replicationStatus.lastSuccess[status.direction] = now;
-        var delay = last ? (now - last) / 1000 : 'unknown';
-        $log.info('Replicate ' + status.direction + ' server successful with ' + delay + ' seconds since the previous successful replication.');
+
+        var now;
```

Comment:
Not possible it will be undefined at line 84, but I agree it's hard to follow the code.  Seems logical to me to keep the value of `now` the same if assigning it in two different places - someone may think it's logical at a later date to compare the values at `lastSuccess` and `lastCompleted`, and my be surprised if they differ.

I'll move the assignment in line with the variable declaration.